pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -315,11 +358,17 @@ fi
 
 tmpDir=$(mktemp -t -d nixos-rebuild.XXXXXX)
 SSHOPTS="$NIX_SSHOPTS -o ControlMaster=auto -o ControlPath=$tmpDir/ssh-%n -o ControlPersist=60"
+if [ -n "$buildHost" -a -n "$flake" ]; then
+    buildHostCmd mkdir "$tmpDir"
```

Comment:
`$tmpDir` is created on the host running `nixos-rebuild`. While the risk of collision is pretty low, I'd still suggest
```suggestion
    buildHostCmd mkdir -p "$tmpDir"
```